DotnetRuntimeMetadata.json

```diff
@@ -1,8 +1,8 @@
 {
   "sdk": {
```

Comment:
Doesn't global.json need to be updated?  Building this branch locally still uses preview.7

Reply:
That's now been updated, by running `.\tools\UpdateDotnetRuntime.ps1`